lib/src/broker/broker_node_provider.dart

```diff
@@ -335,36 +350,62 @@ class BrokerNodeProvider extends NodeProviderImpl implements ServerLinkManager {
   loadUserNodes() async {
     File connsFile = new File("usernodes.json");
     try {
-      String data = await connsFile.readAsString();
-      Map m = DsJson.decode(data);
+      Map m;
+
+      if (loadHandler != null) {
```

Comment:
This part looks duplicated. Could be refactored maybe? I'm not sure if that's make things simpler/cleaner. I'd look into it if I were you.